library/src/main/java/com/joom/grip/FromConfigurator.kt

```diff
@@ -19,15 +19,19 @@ package com.joom.grip
 import com.joom.grip.mirrors.ClassMirror
 import com.joom.grip.mirrors.Type
 import java.io.File
+import java.nio.file.Path
 
 interface FromConfigurator<M, R> {
   infix fun from(classMirrorSource: ClassMirrorSource): QueryConfigurator<M, R>
+
+  @Deprecated("Deprecated.", ReplaceWith("fromPaths(files.map{ it.toPath() })"))
```

Comment:
Why deprecating `File` overloads when `Path` can be easily obtained from `File`?